composable_kernel/include/tensor_operation/reduction_functions_blockwise.hpp

```diff
@@ -23,247 +23,161 @@
  * SOFTWARE.
  *
  *******************************************************************************/
-#ifndef CK_REDUCTION_FUNCTIONS_BLOCKWISE_HPP
-#define CK_REDUCTION_FUNCTIONS_BLOCKWISE_HPP
+#ifndef CK_REDUCTION_FUNCTIONS_PARTITIONED_BLOCKWISE_HPP
```

Comment:
filename and Macro name are not consistent

Reply:
Yes